README.md

```diff
@@ -80,17 +85,29 @@ according to your preferences.
 ### Configurable parameters
 
 **Maven** :
-<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` commands to resolve dependencies for
+<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` command to resolve dependencies for
 Maven projects.
 Path of the `JAVA_HOME` directory is required by the `mvn` executable.
 If the paths are not provided, your IDE's `PATH` and `JAVA_HONE` environments will be used to locate the executables.
 
-**Npm** :
-<br >Path of the `npm` executable allows Exhort to locate and execute `npm` commands to resolve dependencies for Node
+**Node** :
+<br >Path of the `npm` executable allows Exhort to locate and execute the `npm` command to resolve dependencies for Node
```

Comment:
Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `npm` executable allows ...`` --> 
`Set the full path of the Node executable, which allows ...`